src/components/Charts/ChartRt.tsx

```diff
@@ -0,0 +1,265 @@
+import { Group } from '@vx/group';
+import { ParentSize } from '@vx/responsive';
+import { scaleLinear, scaleTime } from '@vx/scale';
+import { GridRows } from '@vx/grid';
+import { curveNatural } from '@vx/curve';
+import { AxisBottom, AxisLeft } from '@vx/axis';
+import { LinePath, Area } from '@vx/shape';
+import { RectClipPath } from '@vx/clip-path';
+import { useTooltip } from '@vx/tooltip';
+import { localPoint } from '@vx/event';
+import { ProjectionDataset, RT_TRUNCATION_DAYS } from '../../models/Projection';
+import { CHART_END_DATE, CASE_GROWTH_RATE, Zones } from '../../enums/zones';
+import BoxedAnnotation from './BoxedAnnotation';
+import HoverOverlay from './HoverOverlay';
+import {
+  formatDecimal,
+  getChartRegions,
+  getTruncationDate,
+  getZoneByValue,
+  last,
+  randomizeId,
+} from './utils';
+import * as Style from './Charts.style';
+
+const computeTickPositions = (minY: number, maxY: number, zones: Zones) => {
+  const maxZones = zones.MEDIUM.upperLimit;
+  const maxTick = maxY < maxZones ? 1.5 * maxZones : maxY;
+  return [minY, zones.LOW.upperLimit, zones.MEDIUM.upperLimit, maxTick];
+};
+
+// Accessing the data
+const x = (d: any) => new Date(d.x);
+const yRt = (d: any) => d?.y?.rt;
+const yHigh = (d: any) => d?.y?.high;
+const yLow = (d: any) => d?.y?.low;
+
+const hasData = (d: any) =>
+  !isUndefined(x(d)) &&
+  !isUndefined(yRt(d)) &&
+  !isUndefined(yLow(d)) &&
+  !isUndefined(yHigh(d));
+
+const ChartRt = ({
+  projectionDataset,
+  width,
+  height = 400,
+  marginTop = 5,
+  marginBottom = 40,
+  marginLeft = 40,
+  marginRight = 5,
+}: {
+  projectionDataset: ProjectionDataset;
+  width: number;
+  height?: number;
+  marginTop?: number;
+  marginBottom?: number;
+  marginLeft?: number;
+  marginRight?: number;
+}) => {
+  const chartWidth = width - marginLeft - marginRight;
+  const chartHeight = height - marginTop - marginBottom;
+
+  const data = projectionDataset.data.filter(hasData);
+
+  // TODO(@pnavarrc): This is so TS doesn't complain
+  const minDate = d3min(data, x) || new Date('2020-01-01');
+  const maxDate = CHART_END_DATE;
+
+  const xScale = scaleTime({
+    domain: [minDate, maxDate],
+    range: [0, chartWidth],
+  });
+
+  const yDataMin = 0;
+  const yDataMax = d3max(data, yRt);
+
+  const yScale = scaleLinear({
+    domain: [yDataMin, yDataMax],
+    range: [chartHeight, 0],
+  });
+
+  const xCoord = (d: any) => xScale(x(d));
+  const yCoord = (d: any) => yScale(yRt(d));
+
+  const yTicks = computeTickPositions(yDataMin, yDataMax, CASE_GROWTH_RATE);
+  const yAxisTicks = _tail(yTicks);
+
+  const { x: lastValidDate } = last(data);
+  const dateTruncation = getTruncationDate(lastValidDate, RT_TRUNCATION_DAYS);
+  const prevData = data.filter((d: any) => x(d) <= dateTruncation);
+  const restData = data.filter((d: any) => x(d) >= dateTruncation);
+  const truncationDataPoint = last(prevData);
+
+  const regions = getChartRegions(yDataMin, yDataMax, CASE_GROWTH_RATE);
```

Comment:
what's the difference betwen a region and a zone and a level?

Reply:
The regions have similar properties than zones, except that they have `valueFrom` and `validTo`, which are calculated from the data and the zone limits. We need those values to calculate the position of the High, Medium, and Low labels and to know which part of the line to give each color.